### DISCUSSION TITLE
Notifications via email using pure SMTP relay server

### DISCUSSION THREAD

**Discussion by robert-ulbrich-mercedes-benz** [flyteorg/flyte#5088] [RFC Incubator]
**Motivation**
Currently notifications by email can only be sent with AWS SES and Sendgrid. Many companies have own SMTP servers that can be used for sending emails instead of the both paid services mentioned before. Unfortunately this is not supported by Flyte. 

**Functional description**

Integrate the capabilities to configure a SMTP server for sending email notifications. Authentication against the SMTP server should be supported as well as a connection encrypted with TLS.

**Comment by kumare3** [comment]
I like this idea too, cc @EngHabu / @eapolinario WDYT?

**Comment by davidmirror-ops** [comment]
2024-04-25 Contributors meetup notes: good to go but as a PR, it doesn't need an RFC. Thanks!
